hadoop-mapreduce-project/bin/mapred

```diff
@@ -105,6 +105,8 @@ function mapredcmd_case
     minicluster)
       hadoop_add_classpath "${HADOOP_YARN_HOME}/${YARN_DIR}/timelineservice"'/*'
       hadoop_add_classpath "${HADOOP_YARN_HOME}/${YARN_DIR}/test"'/*'
+      junitjar=$(echo "${HADOOP_TOOLS_LIB_JARS_DIR}"/junit-[0-9]*.jar)
```

Comment:
I think echo is not required and the following is sufficient:
```
junitjar="${HADOOP_TOOLS_LIB_JARS_DIR}/junit-[0-9]*.jar"
```

testing:
```
bash-5.0$ arg1="foo bar"
bash-5.0$ echo "$arg1"
foo bar
bash-5.0$ arg2="${arg1}/junit-[0-9]*.jar"
bash-5.0$ echo "$arg2"
foo bar/junit-[0-9]*.jar
```

`${junitjar}` must be double-quoted.

Reply:
Sorry, echo is required to resolve the jar file.

Reply:
We need shell filename expansion before adding the path to CLASSPATH. Java only accepts `/path/to/*` wildcard.